README.md

```diff
@@ -101,7 +103,7 @@ class CardIOExample extends Component {
 
   render() {
     return (
-      <View>
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
permit to center the button. On Iphone XS the text is almost not visible because it's at top border of the phone 
![image](https://user-images.githubusercontent.com/749374/49947508-b7431380-fef1-11e8-9634-7a6b7f20a42d.png)